app/main.py

```diff
@@ -1 +1,32 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False
+                 ) -> None:
```

Comment:
Place `self` on a new line as well. If your closing parenthesis does not have any arguments on the same line, your opening parenthesis should not have any arguments on the same line.